configs/selfsup/maskfeat/maskfeat_vit-base-p16_8xb256-amp-coslr-300e_in1k.py

```diff
@@ -0,0 +1,48 @@
+_base_ = [
+    '../_base_/models/maskfeat_vit-base-p16.py',
+    '../_base_/datasets/imagenet_maskfeat.py',
+    '../_base_/schedules/adamw_coslr-300e_in1k.py',
+    '../_base_/default_runtime.py',
+]
+
+# optimizer wrapper
+optimizer = dict(lr=2e-4 * 8, betas=(0.9, 0.999), weight_decay=0.05)
+optim_wrapper = dict(
+    type='AmpOptimWrapper',
+    loss_scale='dynamic',
+    optimizer=optimizer,
+    paramwise_cfg=dict(
+        norm_decay_mult=0.0,
+        bias_decay_mult=0.0,
+        custom_keys={
+            # 'pos_embed': dict(decay_mult=0.),
```

Comment:
Please explain why ``pos_embed`` and  ``cls_token`` should be decayed. Better provide some coments, reasoning the philosophy behind such design.